### ISSUE TITLE
Full node got Disconnected since last night!

### ISSUE THREAD

**Issue by afromanplay** [Chia-Network/chia-blockchain#4949]
HI , guys do u have any idea why its still farming but stays " not synced " ?I can't figure it out ! im running on full node and i have plenty of plots running so it will be a huge wasting time if want to reset the gui...... any suggestion ?
chia v1.1.5 ,GUI
win 10
![118257611-1df7fd00-b4af-11eb-901f-de7575500324](https://user-images.githubusercontent.com/54713396/118259962-1128d880-b4b2-11eb-9ef3-bb0338c55310.jpg)
![118257647-27816500-b4af-11eb-96b3-6a49908cf825](https://user-images.githubusercontent.com/54713396/118259967-11c16f00-b4b2-11eb-9314-e2abb479da28.jpg)
![118257660-2b14ec00-b4af-11eb-8975-f35250512a80](https://user-images.githubusercontent.com/54713396/118259972-125a0580-b4b2-11eb-9219-5db3e9611939.jpg)

2021-05-14T12:23:01.126 full_node full_node_server : INFO Cannot connect to host 167.114.1.167:8444 ssl:<ssl.SSLContext object at 0x000001F2E0C779E8> [Connect call failed ('167.114.1.167', 8444)]
2021-05-14T12:23:05.786 full_node full_node_server : INFO Cannot connect to host 167.114.1.167:8444 ssl:<ssl.SSLContext object at 0x000001F2806D2F28> [Connect call failed ('167.114.1.167', 8444)]
2021-05-14T12:23:11.006 full_node full_node_server : INFO Cannot connect to host 167.114.1.167:8444 ssl:<ssl.SSLContext object at 0x000001F2E4EFE048> [Connect call failed ('167.114.1.167', 8444)]
2021-05-14T12:23:15.613 full_node full_node_server : INFO Cannot connect to host 167.114.1.167:8444 ssl:<ssl.SSLContext object at 0x000001F2E4EFE588> [Connect call failed ('167.114.1.167', 8444)]
2021-05-14T12:23:20.225 full_node full_node_server : INFO Cannot connect to host 167.114.1.167:8444 ssl:<ssl.SSLContext object at 0x000001F281119DD8> [Connect call failed ('167.114.1.167', 8444)]
2021-05-14T12:23:25.170 full_node full_node_server : INFO Cannot connect to host 167.114.1.167:8444 ssl:<ssl.SSLContext object at 0x000001F2878312E8> [Connect call failed ('167.114.1.167', 8444)]
ps , Port 8444 is open for me

**Comment by stipus** [comment]
Is your chia database (.chia/mainnet/db directory) on a spinning hard drive ?

**Comment by afromanplay** [comment]
its on on a separate ssd , and it doesn't have that much read n write !
![sdfsdfsd](https://user-images.githubusercontent.com/54713396/118261513-f0618280-b4b3-11eb-9f8b-f043caba24ae.jpg)

**Comment by stipus** [comment]
OK. Thank you. This is not linked to the bug I reported (having the database on a spinning disk).

**Comment by afromanplay** [comment]
> OK. Thank you. This is not linked to the bug I reported (having the database on a spinning disk).

so? what to do?!

**Comment by afromanplay** [comment]
can i make it get synced again without restarting it ? or just reset the GUI without having the plot stopped ??

**Comment by stipus** [comment]
If you are using the GUI to plot, I'm afraid your plots will be interrupted if you close the GUI.
In the future, you may want to use the CLI to plot.

**Comment by afromanplay** [comment]
> If you are using the GUI to plot, I'm afraid your plots will be interrupted if you close the GUI.
> In the future, you may want to use the CLI to plot.

so u say that i have to reset the GUI??

**Comment by Plopsi** [comment]
Does you node has any connections? Try to connect ot one manually For example to the introducer (that's basically it's purpose, sending peers around) or just grab on from https://chia.keva.app

**Comment by afromanplay** [comment]
> Does you node has any connections? Try to connect ot one manually For example to the introducer (that's basically it's purpose, sending peers around) or just grab on from https://chia.keva.app

i think it has ! 
![sdfsdfsd](https://user-images.githubusercontent.com/54713396/118267208-f9565200-b4bb-11eb-9413-1913298e58bc.jpg)
im trying to connect

**Comment by afromanplay** [comment]
> Does you node has any connections? Try to connect ot one manually For example to the introducer (that's basically it's purpose, sending peers around) or just grab on from https://chia.keva.app

i think it has !
![rterterter64674747](https://user-images.githubusercontent.com/54713396/118267308-2276e280-b4bc-11eb-8b34-222309822148.jpg)
im trying to connect

**Comment by afromanplay** [comment]
i added this one  84.127.228.247 | 8444/8444 now it starts to sync ! 




</body>
</html>

**Comment by IIPoliII** [comment]
> 
> 
> Does you node has any connections? Try to connect ot one manually For example to the introducer (that's basically it's purpose, sending peers around) or just grab on from https://chia.keva.app

Awesome that fixed my syncing issues

**Comment by afromanplay** [comment]
is that ok ?
![rterterter64674747](https://user-images.githubusercontent.com/54713396/118268041-2820f800-b4bd-11eb-9dcf-433b0f8ac19c.jpg)
![118257660-2b14ec00-b4af-11eb-8975-f35250512a80](https://user-images.githubusercontent.com/54713396/118268096-3a9b3180-b4bd-11eb-843f-831327e5bcff.jpg)

**Comment by IIPoliII** [comment]
What the fuck I got desynced again....

**Comment by afromanplay** [comment]
me as well!

**Comment by afromanplay** [comment]
i dont get it

**Comment by IIPoliII** [comment]
Do we need to touch something on the windows Firewall? Or route the port 8444 maybe?

**Comment by Plopsi** [comment]
No when you can connect it's fine - The problem is when you only have one peer and it drops, you stand there without any further once, try to connect to 3-4 initially. Within an hour you *should* have a couple more if everything works out right and the other peers share valid nodes

**Comment by IIPoliII** [comment]
Well when I add a node to connect too it simply disapears...

**Comment by Plopsi** [comment]
if you tried to often you are banned a while from a node. Also if you have auth/cert problems - try to restart chia.

**Comment by IIPoliII** [comment]
Well I have another node running on linux and it works perfectly, I forwarded now my port after seeing this https://github.com/Chia-Network/chia-blockchain/wiki/Resolving-Sync-Issues---Port-8444
But even if I sync with one of my own nodes I cannot sync.
What is the ban time for a node?

**Comment by Plopsi** [comment]
don't know i was also banned from one of my local ones :D (saw it in their log) - but felt asleep and tried 2-3h later and it worked again.

**Comment by JoJ123** [comment]
@Plopsi I also had the "local" banned stuff. Still I can't sync even if I have more than 20 nodes connected...

**Comment by Plopsi** [comment]
can you try one of my current list

188.195.181.40:8444 does this one work?

or

104.3.39.94:8444

or let's see...

75.85.172.191:8444

this one.

That are the top 3 Mbit Down from my current conection list.

**Comment by IIPoliII** [comment]
Can you try : 
5.9.38.90:8444 (DE)
51.154.165.162:8444 (CH)
51.154.10.127:8444 (CH)

These are my machines the Swiss machines are connected at 10 gb/s and the german one at 1gb/s let me know if it works

**Comment by IIPoliII** [comment]
> 
> 
> can you try one of my current list
> 
> 188.195.181.40:8444 does this one work?
> 
> or
> 
> 104.3.39.94:8444
> 
> or let's see...
> 
> 75.85.172.191:8444
> 
> this one.
> 
> That are the top 3 Mbit Down from my current conection list.

I think the GUI is shit it doesn't connect at all.... I was able to connect my machines but only in local? Wut? I forwarded all the ports needed for it.

**Comment by Plopsi** [comment]
then try CLI ^^ https://github.com/Chia-Network/chia-blockchain/wiki/CLI-Commands-Reference It's also a little bit more noise about what is has done

`chia show -a <ip>:<port>` is the connect to peer command

**Comment by Plopsi** [comment]
> 
> 
> Can you try :
> 5.9.38.90:8444 (DE)
> 51.154.165.162:8444 (CH)
> 51.154.10.127:8444 (CH)
> 
> These are my machines the Swiss machines are connected at 10 gb/s and the german one at 1gb/s let me know if it works
> @Plopsi

Nice, just connected. I keep an I on the connections the next hour to see if one get's dropped by chia or connection stay stable.

**Comment by sbstockmeup** [comment]
Keeps happening. Longest I was unsync'd was 2 hours before I had to restart GUI. This morning it happened again for at least 20 mins.

**Comment by Plopsi** [comment]
@sbstockmeup maybe 24h disconnect? just guessing.

**Comment by burak-dereli** [comment]
![Adsız](https://user-images.githubusercontent.com/84191040/118285250-efdbe280-b4d9-11eb-83b7-49e7eb370a1f.jpg)

I'm using Ubuntu 20.04 and it hasn't synced for 28 hours. 28 ip addresses are connected right now. These include node.chia.net. Port 8444 is also open. it synchronizes immediately to another windows computer on the same network. I did not understand the problem. I have added all the ip addresses in https://chia.keva.app/ and here. But it always says Synchronizing. I'll go crazy in the end.

**Comment by Tongie969** [comment]
> If you are using the GUI to plot, I'm afraid your plots will be interrupted if you close the GUI.
> In the future, you may want to use the CLI to plot

No, just click on keys that will reload but not stop plotting.

**Comment by Tongie969** [comment]
https://github.com/martomi/chiadog

**Comment by Plopsi** [comment]
@Tongie969 interresting, will take a look

Sorry to all other I'm currently out of knowledge, ideas, guesses, prayers and thoughts...
I run chia-docker on a tiny little NAS and had nearly zero problems... I know it's pain and when I have an idea I come back and tell you... but for now... "brain.exe has stopped working"...

I really wish it would work on first try at everybodies system... the more the better to compete the pools.. but... :(

**Comment by dvd101x** [comment]
I got the same issues, exactly as stated in this thread, following all suggestions.

**Comment by zeybor** [comment]
I am also getting the very same issues. even-though I am connected to ~50 peers after a few hours of running without issues, i suddenly get out of sync. I try to restart the node through the CLI, close & re-open the GUI. but nothing seems to work. The only solution appears to be restarting the machine which is really annoying, because I am also plotting on this device. Today I could not complete a single plot, because I had to restart the machine twice. 

the only error code i see in logs is:

`2021-05-14T18:48:21.287 full_node full_node_server        : ERROR    Exception:  <class 'concurrent.futures._base.CancelledError'>, closing connection {'host': '127.0.0.1', 'port': 8449}. Traceback (most recent call last):
  File "chia\server\server.py", line 531, in api_call
  File "asyncio\tasks.py", line 435, in wait_for
concurrent.futures._base.CancelledError`

**Comment by nothinglastsforever** [comment]
I see I'm not the only one having this problem, @Plopsi. 24+ hours and it hasn't moved. Obviously something is very broken if this many people are having issues.

**Comment by IIPoliII** [comment]
> 
> 
> > Can you try :
> > 5.9.38.90:8444 (DE)
> > 51.154.165.162:8444 (CH)
> > 51.154.10.127:8444 (CH)
> > These are my machines the Swiss machines are connected at 10 gb/s and the german one at 1gb/s let me know if it works
> > @Plopsi
> 
> Nice, just connected. I keep an I on the connections the next hour to see if one get's dropped by chia or connection stay stable.

What.... 51.154.165.162:8444 desynced again the others are stable but why?

**Comment by IIPoliII** [comment]
For my side everytime I manually connect to someone it doesn't appear...

**Comment by ikthayt** [comment]
I also have been having a lot of issues syncing and not gaining height, but seem to be connected to a full complement of nodes.  But i also noticed that when I did a `chia show -s -c`  that I had a LOT of connections that were very low height and more than a handful that were just zeros.  I decided to drop them from my list.  

From the help command (i.e., chia show -h):

> Usage: chia show [OPTIONS]Options:  -p, --rpc-port INTEGER          Set the port where the Full Node is hosting                                  the RPC interface. See the rpc_port under                                  full_node in config.yaml
>
>  -wp, --wallet-rpc-port INTEGER  Set the port where the Wallet is hosting the
>                                  RPC interface. See the rpc_port under wallet
>                                  in config.yaml
>
>  -s, --state                     Show the current state of the blockchain
>  -c, --connections               List nodes connected to this Full Node
>  -e, --exit-node                 Shut down the running Full Node
>  -a, --add-connection TEXT       Connect to another Full Node by ip:port
>  -r, --remove-connection TEXT    Remove a Node by the first 8 characters of
>                                  NodeID
>
>  -bh, --block-header-hash-by-height TEXT
>                                  Look up a block header hash by block height
>  -b, --block-by-header-hash TEXT
>                                  Look up a block by block header hash
>  -h, --help                      Show this message and exit.

For example:
`chia show -r xyz123ab`    

You just copy the codes from the list created by the `show -s -c` command and enter them in.  I dropped all my nodes that were zero and it started climbing again.  I think there is a bug with zeros (or there are some shenanigans going on and people are purposefully blocking syncs).

I leveraged excel to make it faster, but can manually do it too.  I'm sure someone else could figure out a smarter/script-ier way to do it, but I'm not that smart.

YMMV, but it got me unstuck for the short term.

Happy farming, friends

**Comment by dvd101x** [comment]
I made a script that adds everything from [https://chia.keva.app](https://chia.keva.app) with the CLI

I'm running this for windows and seems to work form me, not ideal but I was desperate:

`.\chia show -a 5.189.201.93:8444;.\chia show -a 101.66.242.200:8444;.\chia show -a 91.5.105.207:8444;.\chia show -a 188.75.237.73:8444;.\chia show -a 222.90.142.21:8444;.\chia show -a 36.25.92.92:8444;.\chia show -a 109.127.49.163:8444;.\chia show -a 100.34.155.23:8444;.\chia show -a 92.234.80.105:8444;.\chia show -a 103.20.120.53:8444;.\chia show -a 180.183.121.18:8444;.\chia show -a 99.250.171.171:8444;.\chia show -a introducer-apne.chia.net:8444;.\chia show -a introducer-apse.chia.net:8444;.\chia show -a introducer-or.chia.net:8444;.\chia show -a introducer-va.chia.net:8444;.\chia show -a introducer-eu.chia.net:8444`

**Comment by burak-dereli** [comment]
![Ekran görüntüsü 2021-05-14 22-14-26](https://user-images.githubusercontent.com/84191040/118318666-5e349b00-b502-11eb-9cdc-5c16ab14280d.png)


I also give such an error. where is ubuntu chia?

**Comment by Plopsi** [comment]
chia is a little hidden.. you need to switch to the right directory first: check https://github.com/Chia-Network/chia-blockchain/wiki/CLI-Commands-Reference

And yes, I'm as annoyed and angry that this.. *piep* *piep* *piep* still are as *piep* to put their *piep* *piep* software *piep* software under a *piep* ... *piep* "Data" folder... *breathing*

This is NOT userfriendly, on a simple Drive-Cleanup chia gets.. yeah.. bye bye... I never understood why someone put *EXECUTABLES* under a *DATA* directory... sry just because the default framework says says so is no excuse.

*breathing**breathing**brathering*

ok sry *anotherbrathering* this annoys me sooooo much over every wallet and every software, not only crypto.

**Comment by IIPoliII** [comment]
> 
> 
> ![Ekran görüntüsü 2021-05-14 22-14-26](https://user-images.githubusercontent.com/84191040/118318666-5e349b00-b502-11eb-9cdc-5c16ab14280d.png)
> 
> I also give such an error " command not found ". where is ubuntu chia?

Which version are you using?
.\Chia works better by doing ./chia and you need to make . . /activate in the chia repo that you cloned locally

**Comment by Plopsi** [comment]
Hmm yeah they forgot linux path.. try something like /bin/chia or /sbin/chia... or don't knowm, maybe they just missed to document this.

Try `find  / . -name chia` hope this command is still valid... and if something returns go to that directory and try again

**Comment by burak-dereli** [comment]
![Adsız jpg2](https://user-images.githubusercontent.com/84191040/118324387-a5bf2500-b50a-11eb-80da-267bbcf6355b.jpg)


@Plopsi  Thanks.
/chia-blockchain/resources/app.asar.unpacked/daemon/chia 
thats here

**Comment by tibbon** [comment]
I'm experiencing similar. I can attempt to manually add them, but then they don't seem to stick, nor does it seem to be automatically adding them. I tried adding an introducer, but that's not helping either!

**Comment by burak-dereli** [comment]
![Adsız jpg1](https://user-images.githubusercontent.com/84191040/118325070-9391b680-b50b-11eb-94c0-0aea34e132e8.jpg)

It Work. I hope it will synchronize as soon as possible

**Comment by Plopsi** [comment]
> 
> 
> I made a script that adds everything from https://chia.keva.app with the CLI
> 
> I'm running this for windows and seems to work form me, not ideal but I was desperate:
> 
> `.\chia show -a 5.189.201.93:8444;.\chia show -a 101.66.242.200:8444;.\chia show -a 91.5.105.207:8444;.\chia show -a 188.75.237.73:8444;.\chia show -a 222.90.142.21:8444;.\chia show -a 36.25.92.92:8444;.\chia show -a 109.127.49.163:8444;.\chia show -a 100.34.155.23:8444;.\chia show -a 92.234.80.105:8444;.\chia show -a 103.20.120.53:8444;.\chia show -a 180.183.121.18:8444;.\chia show -a 99.250.171.171:8444;.\chia show -a introducer-apne.chia.net:8444;.\chia show -a introducer-apse.chia.net:8444;.\chia show -a introducer-or.chia.net:8444;.\chia show -a introducer-va.chia.net:8444;.\chia show -a introducer-eu.chia.net:8444`

wow... nice one.. Go and upthum (at least) post https://github.com/Chia-Network/chia-blockchain/issues/4949#issuecomment-841440141

This is a nice alternative to manually searching peers and don't find them.. AND the not, or spacial working auto-peer discovery.
Hell that's my mos prolem, without solution like  @dvd101x at https://github.com/Chia-Network/chia-blockchain/issues/4949#issuecomment-841440141 chia connect would be not only slow but ver slow to be not to insulting.
----
@bramcohen You were deeply involved to BitTorrent, which has a working peer-discovery. Go get some sleep and check what's peeping up here. I think (25-33% cutoff pe rciv) is ok for now, as long as the network kept safe,

**Comment by skinski** [comment]
Workaround that works for me every time:

I'm not sure where the databases are held on the Linux version however:

Stop Chia

Rename or delete: .chia\mainnet\db\peer_table_node.sqlite
Note: On windows this is found under your profile (e.g. c:\users%username%) by default.

Start Chia

This forces the repopulation of the peer database with newly scanned/found nodes and removes old nodes that may no longer exist.

As I understand it, the peers that you had connected to previously were probably temporary plotter nodes and likely all removed over time as plots were migrated to farmer nodes. So in essence you end up with a list of dead nodes over time.

I had read an article where someone had suggested renaming the entire .Chia folder however, this forces a resync from scratch due to also removing the main database. Simply renaming the peer table database (as above) retains the main database and so you don't have to start syncing from 0 again, it works pretty well for me.

Maybe devs could add a validation routine that removes stale node entries from the peer database and forces a rescan once number of active nodes diminishes by a certain amount?

Also - make sure you upgrade to the latest version.

**Comment by fabioyy** [comment]
reached the same conclusion then skinski.

i delete all tables of peer_table_node.sqlite.
so far. is syncing again.

**Comment by zeybor** [comment]
> Rename or delete: .chia\mainnet\db\peer_table_node.sqlite
> Note: On windows this is found under your profile (e.g. c:\users%username%) by default.

I have deleted the peer_table_node.sqlite. A new file was generated after launch, and I got synced for a few minutes, and then I got unsynced again.

**Comment by skinski** [comment]
> > Rename or delete: .chia\mainnet\db\peer_table_node.sqlite
> > Note: On windows this is found under your profile (e.g. c:\users%username%) by default.
> 
> I have deleted the peer_table_node.sqlite. A new file was generated after launch, and I got synced for a few minutes, and then I got unsynced again.

Give it a while longer mate - it will sync

**Comment by zeybor** [comment]
> Give it a while longer mate - it will sync
> 
> I'm guessing that there's a high amount of plotter nodes dropping off the network frequently due to being decommissioned, replaced and rebuilt as users start to understand the plotting mechanics and optimise their hardware, or have simply finished plotting and migrated their plots to lesser hardware to farm.

Thanks for getting back, this might well be the case, but shouldn't many more people experience this issue?

Waited about an hour now, and the GUI is still showing me out of sync.

From what I understand, I occasionally get synced at some point, as I see some farming attempts in the GUI, but the last one was 20 minutes ago. (But nothing in the logs?) I can also see some improvement between heights on ` Chia show -s` as seen in the screenshot.

In the logs I see some errors like these:

`2021-05-15T09:53:47.522 full_node full_node_server        : ERROR    Exception:  <class 'concurrent.futures._base.CancelledError'>, closing connection {'host': '188.36.43.239', 'port': 8444}. Traceback (most recent call last):
  File "chia\server\server.py", line 531, in api_call
  File "asyncio\tasks.py", line 435, in wait_for
concurrent.futures._base.CancelledError`

`2021-05-15T09:54:51.754 full_node full_node_server        : ERROR    Exception:  <class 'concurrent.futures._base.TimeoutError'>, closing connection {'host': '125.77.93.148', 'port': 8444}. Traceback (most recent call last):
  File "chia\server\server.py", line 531, in api_call
  File "asyncio\tasks.py", line 449, in wait_for
concurrent.futures._base.TimeoutError`

`2021-05-15T09:54:51.770 full_node asyncio                 : ERROR    Task exception was never retrieved
future: <Task finished coro=<ChiaServer.incoming_api_task.<locals>.api_call() done, defined at chia\server\server.py:482> exception=CancelledError()>
concurrent.futures._base.CancelledError`

`2021-05-15T10:36:43.422 full_node full_node_server        : ERROR    Exception: Failed to fetch block 283593 from None, timed out, None. Traceback (most recent call last):
  File "chia\server\server.py", line 521, in wrapped_coroutine
  File "chia\full_node\full_node_api.py", line 104, in new_peak
  File "chia\full_node\full_node.py", line 361, in new_peak
  File "chia\full_node\full_node.py", line 310, in short_sync_backtrack
  File "chia\full_node\full_node.py", line 295, in short_sync_backtrack
ValueError: Failed to fetch block 283593 from None, timed out`


![Screenshot_2](https://user-images.githubusercontent.com/83019631/118353167-ca001d80-b56d-11eb-932b-5b823355d179.jpg)
![Screenshot_3](https://user-images.githubusercontent.com/83019631/118353168-ca98b400-b56d-11eb-8f2e-9ea78b50e736.jpg)
![Screenshot_4](https://user-images.githubusercontent.com/83019631/118353170-cb314a80-b56d-11eb-86e3-673e6713da03.jpg)
![Screenshot_5](https://user-images.githubusercontent.com/83019631/118353171-cb314a80-b56d-11eb-8be6-dd7683b5b13c.jpg)

**Comment by afromanplay** [comment]
I simply deleted the Chia and added peers manually but now i found this in my log
2021-05-15T01:17:09.655 full_node chia.full_node.full_node: ERROR    got weight proof request for unknown peak 98daa97e44aaffbd2fe3c51c9483c53d034a1c613868c611ecaa48f8b40e082a
2021-05-15T01:23:09.693 full_node chia.full_node.full_node: ERROR    got weight proof request for unknown peak d90f3f86d9807a9e1c30ca80fa59a175f27b8b4b81b853a80d0427f83a4c6e19
2021-05-15T01:29:09.948 full_node chia.full_node.full_node: ERROR    got weight proof request for unknown peak 7671a62815db0fc1e7b5740f8c51c77e62fb49f1007f5b5b08d8a7e42cf70544
2021-05-15T01:35:09.986 full_node chia.full_node.full_node: ERROR    got weight proof request for unknown peak a4e7a715efcbb0af57fc4d503066e392a6b0fa9a82a61d2aac7b85c8be6b29de
2021-05-15T01:41:10.016 full_node chia.full_node.full_node: ERROR    got weight proof request for unknown peak 068e6147fa501805bb1badc82a10bb1cc8b5c5f72cf9012b661102c4c10b9df1
2021-05-15T01:47:10.061 full_node chia.full_node.full_node: ERROR    got weight proof request for unknown peak ecd3705c0fece918bf2521fd591966a8d0b23841544e8e44aa045882a34bc1c2
2021-05-15T01:53:10.100 full_node chia.full_node.full_node: ERROR    got weight proof request for unknown peak 00a87ed4c7d80bd7fb3b6f149d228c61b0be9a6271e6cdd8e7bffb1483dc210d
2021-05-15T01:59:10.137 full_node chia.full_node.full_node: ERROR    got weight proof request for unknown peak 04318f9a9fd8a8b6aed3e4417f91e99d25b5b9ae69fc36e02050ebb536e23b37
2021-05-15T02:05:10.168 full_node chia.full_node.full_node: ERROR    got weight proof request for unknown peak 7c4f32988965ef8791b82e43cd5de20b29f41796fee8363ef2941dd4d20d6e53
2021-05-15T02:11:10.220 full_node chia.full_node.full_node: ERROR    got weight proof request for unknown peak 16b557df83c41e51e0f93040584106753de6423bc1aecae45a28cac8729d48ba
2021-05-15T02:17:10.256 full_node chia.full_node.full_node: ERROR    got weight proof request for unknown peak 39852a934562cd8d07d9ef43cab14ff69f6e8bbb0d4f9111af3c6f4fd73fd506
2021-05-15T02:23:10.303 full_node chia.full_node.full_node: ERROR    got weight proof request for unknown peak d82ee450cc0331fc12a3445f5323ab29667031dfd1aad6c6f2b1413c2b1f79e9

should i be woory about it?

**Comment by skinski** [comment]
You could try to remove the plot directory where the chia plots were located from the configuration (note:  I do NOT mean delete the actual folder) and your key (back up mnemonic passphrase first) and then restart and add them both again.  Not sure but this may well remove the defective plots that you have deleted - they must still be referenced somewhere...

From the client - go to plots, add plots and click on the three dots to the right of the + Add a Plot button.
Click on Add Plot Directory from drop down
From this screen you can remove existing Plot Directories (click on the bin icon next to the directory you want to remove)

It may be enough at this point to just add the Plot directory again, I'm not sure.  I would at least try a restart of the chia client in-between.

You could also try combining this with adding and removing the key using the passphrase to restore:
Remove folder
Remove Key
Add Key
Add folder

I'm not sure if any of these will actually work as I have no knowledge of the code behind the GUI and whether it will retain reference to the missing plots.

The same actions can be carried out at the command line if you  know what you are doing - search Chia CLI reference in Google.

**Comment by zeybor** [comment]
> > Give it a while longer mate - it will sync
> > I'm guessing that there's a high amount of plotter nodes dropping off the network frequently due to being decommissioned, replaced and rebuilt as users start to understand the plotting mechanics and optimise their hardware, or have simply finished plotting and migrated their plots to lesser hardware to farm.
> 
> Thanks for getting back, this might well be the case, but shouldn't many more people experience this issue?
> 
> Waited about an hour now, and the GUI is still showing me out of sync.
> 
> From what I understand, I occasionally get synced at some point, as I see some farming attempts in the GUI, but the last one was 20 minutes ago. (But nothing in the logs?) I can also see some improvement between heights on ` Chia show -s` as seen in the screenshot.

I had to restart my pc, as nothing seemed to work. After the restart, everything works fine. I am synched, and all harvesters are farming. I guess until it'll stop again later on today.... So this appears to be a windows issue? As the only fix appears to be a restart.

**Comment by fabioyy** [comment]
changed the .db to  ssd to see if latency was the problem.. but i still  got timeouts errors.
one thing that i notice. there is an Python  process ( full node ) using 100% of a Single core cpu.

**Comment by mdp19900707** [comment]
> Can you try :
> 5.9.38.90:8444 (DE)
> 51.154.165.162:8444 (CH)
> 51.154.10.127:8444 (CH)
> 
> These are my machines the Swiss machines are connected at 10 gb/s and the german one at 1gb/s let me know if it works
> @Plopsi

Thank you !   very  very  very ...

**Comment by kmoore134** [comment]
> changed the .db to ssd to see if latency was the problem.. but i still got timeouts errors.
> one thing that i notice. there is an Python process ( full node ) using 100% of a Single core cpu.

Same issue here. When it gets permanently de-synced I notice that full node process spinning at 100% of a core.

**Comment by IIPoliII** [comment]
> > changed the .db to ssd to see if latency was the problem.. but i still got timeouts errors.
> > one thing that i notice. there is an Python process ( full node ) using 100% of a Single core cpu.
> 
> Same issue here. When it gets permanently de-synced I notice that full node process spinning at 100% of a core. 

Also founded out I got desynced when I download some off my other plots but on another disk. Maybe this process checks the plots, all the time during the transfer?

**Comment by skinski** [comment]
Observation - Both of my PC's that are currently holding plots have disconnected but only once or twice, I removed the peer node database and restarted, they seem to pickup new nodes and sync over a period of around 20-30 minutes from starting.

I did run a chia plot check on both nodes and found bad plots.  I first removed the plot folder from Chia config using the GUI method (CLI should work just as well), then deleted the bad plots and then re-added the folder.  This didn't need me to restart (I'm running Windows OS), I then re-plotted to consume the space from the deleted bad plots and re-checked all plots until all were healthy.

Time will tell if I still get disconnects now and again however, I don't think I'm having as many issues as some in this thread.
